src/server/util/authenticationHelpers.js

```diff
@@ -1,24 +1,79 @@
 const { Pool } = require('pg');
 const { sql } = require('./sqlHelpers');
-const { saltHashPassword } = require('./pwHash');
+const { saltHashPassword, validateHashPassword } = require('./pwHash');
 
 const pool = new Pool();
 
-const getUser = async function(req, res, next) {
-    const { username, email } = req.body;
+const getUserInDatabase = async function(username, email) {
     const client = await pool.connect();
-
     const result = await client.query(sql`SELECT * FROM users WHERE email = ${email} OR username = ${username};`);
     client.release();
+    return result.rows;
+}
+
+const userExistsInDatabase = async function(username, email) {
+    const rows = await getUserInDatabase(username, email);
+    let user, exists = rows.length > 0;
```

Comment:
Nit/in my opinion, a more consistent way to design this API would be to set `user` initially equal to null. Then you can just return `user` and do a if-statement check on it to see if it exists. Then you don't need to return `exists` as well.

Reply:
@ktbee I like this. I've implemented this